setup.cfg

```diff
@@ -42,7 +42,7 @@ install_requires =
     requests>=2.22
     scipy>=1.1.0
     spherical-geometry>=1.2.20
-    stcal>=0.2.2
+    stcal@git+https://github.com/spacetelescope/stcal
```

Comment:
stcal 0.2.3 is out now, so let's change this to

```
stcal>=0.2.3
```

After that, ready to merge.